srcs/config/analyser/ParserStatelessSet.cpp

```diff
@@ -49,12 +50,12 @@ bool Parser::StatelessSet::ParseIpAddressPort_(const std::string &input,
 		}
 		addressStr = addTmp.c_str();
 	}
-	const in_addr_t address = inet_addr(addressStr);
-	if (address == static_cast<in_addr_t>(-1)) {
+	int ret = inet_pton(AF_INET, addressStr, &addr);
+	if (errno || ret != 1) {
```

Comment:
Here we could initialize the errno to 0 before calling inet_pton. In case it had a non-zero value, and the function succeeds.